RULES.md

```diff
@@ -419,3 +420,12 @@ Even though `feed_info.start_date` and `feed_info.end_date` are optional, if one
 ### TooFastTravelNotice
 
 As implemented in the original [Google Python GTFS validator](https://github.com/google/transitfeed/wiki/FeedValidator), the calculated speed between stops should not be greater than 150 km/h (42 m/s SI or 93 mph). 
+
+<a name="AttributionWithoutRoleNotice"/>
+
+### AttributionWithoutRoleNotice
+
+At least one of the fields is_producer, is_operator, or is_authority should be set at 1.
```

Comment:
To be consistent the fields should have the tick marks to render them as code - `is_producer`.